R/markdown-link.R

```diff
@@ -11,18 +11,27 @@
 #' ```
 #' MARKDOWN           LINK TEXT  CODE RD
 #' --------           ---------  ---- --
-#' [fun()]            fun()       T   \\link[=fun]{fun()}
-#' [obj]              obj         F   \\link{obj}
+#' [fun()]            fun()       T   \\link[=fun]{fun()} or
+#'                                    \\link[pkg::file]{fun()}
```

Comment:
```suggestion
#'                                    \\link[pkg:file]{fun()}
```

Similarly elsewhere?